LEAF_Nexus/templates/view_group.tpl

```diff
@@ -127,6 +129,23 @@ function announceAction(actionName) {
     $('#buttonClick').attr('aria-label', 'clicked ' + actionName);
 }
 
+function viewHistory(){
+    dialog_message.setContent('');
```

Comment:
Add the title to read "Acess Group History" dialog.setTitle('Acess Group History');